src/AaveV2FreezeReserves_20230619/DeployV2Freeze.s.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+import {EthereumScript} from 'aave-helpers/ScriptUtils.sol';
+import {GovHelpers} from 'aave-helpers/GovHelpers.sol';
+import {AaveV2FreezeReserves_20230619} from 'src/AaveV2FreezeReserves_20230619/AaveV2FreezeReserves_20230619.sol';
+
+contract DeployPayload is EthereumScript {
+  function run() external broadcast {
+    new AaveV2FreezeReserves_20230619();
+  }
+}
+
+contract CreateProposal is EthereumScript {
+  function run() external broadcast {
+    GovHelpers.Payload[] memory payloads = new GovHelpers.Payload[](1);
+    payloads[0] = GovHelpers.buildMainnet(
+      address(0) // TODO: Replace by actual payload
+    );
+    GovHelpers.createProposal(
+      payloads,
+      GovHelpers.ipfsHashFile(
+        vm,
+        'src/AaveV2FreezeReserves_20230619/V2-to-V3-MIGRATION-NEXT-STEPS.md',
+        // if you set `upload` to `true`, your env must contain PINATA_KEY & PINATA_SECRET
+        // the file will be uploaded automatically once merged to main
+        false
```

Comment:
```suggestion
```
better omit this flag - we just added it for ppl not using this repository, so it's not really needed when doing a pr here.